otter/test/util/test_pure_http.py

```diff
@@ -19,15 +19,16 @@
     add_content_only, add_error_handling, add_json_response,
     add_json_request_data)
 from otter.util.http import APIError
-from otter.test.utils import stub_pure_response, StubResponse, StubTreq
+from otter.test.utils import stub_pure_response, StubResponse, StubTreq, resolve_stubs
+from otter.effect_dispatcher import get_dispatcher
 
 
-Constant = lambda x: StubIntent(ConstantIntent(x))
+ESConstant = lambda x: Effect(Stub(Constant(x)))
```

Comment:
This is pretty useful for other tests too. How about putting it in `test.utils.py`?